src/app/routes/index.test.jsx

```diff
@@ -533,13 +550,17 @@ it('should route to and render a 404 error page', async () => {
 
 it('should render a 404 error page if a data fetch responds with a 404', async () => {
   const pathname = '/pidgin/articles/cwl08rd38p6o';
-  fetchMock.mock(`http://localhost${pathname}.json`, 404);
+  const bffFetchSpy = jest.spyOn(fetchPageData, 'default');
+
+  bffFetchSpy.mockRejectedValue({ message: 'Not found', status: 404 });
```

Comment:
This is way beyond my comprehension now. You are a magician

Reply:
From the comment above, the `spyOn` function lets us mock the return responses from whatever function we are 'spying' on. In this case, we are forcing a `404` using the `mockRejectedValue` function. Behind the scenes, this creates a `Promise.reject`, which forces the code in the BFF fetch to hit the `catch` part of our `try/catch`, and so returns an error response.